README.rst

```diff
@@ -107,7 +107,8 @@ how to create a bucket.
     blob = bucket.get_blob('/remote/path/to/file.txt')
     print blob.download_as_string()
     blob.upload_from_string('New contents!')
-    bucket.upload_file('/remote/path/storage.txt', '/local/path.txt')
+    blob2 = storage.Blob('/remote/path/storage.txt', bucket)
```

Comment:
If we're keeping with "the one way", wouldn't it make more sense to have:

```python
blob = bucket.get_blob('...)  # Get a reference to an existing blob.
blob = bucket.new_blob('...')  # Get a reference to a blob to be created. 
# or
blob = bucket.blob('...')

# and then
blob.upload_from_filename('...')
```

Seems weird to me that for an existing one we do `bucket.get_blob(name)` but for a new one we do `storage.Blob(name, bucket)`...